### DISCUSSION TITLE
Routing questions

### DISCUSSION THREAD

**Discussion by hoegertn** [Hawxy/cdk-tailscale-bastion#4] [Ideas]
I really like the idea as this is exactly what I am looking for.

Two questions:

1) Any reason you do not accept incoming routes from tailscale from another subnet router `--accept-routes`

2) Would it be possible to auto-create route table entries for given remote networks specified in the props?

Happy to create a PR if you like this.

**Comment by Hawxy** [comment]
1. This was initially built as a 1:1 reproduction of the RDS guide so anything not included there was left out. Feel free to add it.
2. Sure, I'm happy to accept a PR.

**Comment by hoegertn** [comment]
Ok, I created #5 to track it

**Comment by hoegertn** [comment]
I am trying to implement this, but my auth key SecretValue is not resolved and the instance is not booting up. According to CFN docs this is not supported yet.

How does this work for you?

**Reply by Hawxy** [reply]
Damn. Whilst I was testing the `SecretValue` support I only used an `unsafePlainText` value which didn't raise this issue. The version of construct I use in my day-to-day is an old version that just passes in a string and thus I never tested the construct with SSM/SecretsManager. 

Feel free to adjust the `authkey` value to a string to unblock yourself and I'll investigate a better solution.

**Reply by hoegertn** [reply]
Using a fix string worked obviously.

What about reading the secret from SM using the AWS CLI instead of a dynamic reference?

**Reply by Hawxy** [reply]
> What about reading the secret from SM using the AWS CLI instead of a dynamic reference?

I was looking into that. I've pushed up a new version with a refactored auth implementation.
